lisa/messages.py

```diff
@@ -137,3 +137,87 @@ class NetworkPPSPerformanceMessage(PerfMessage):
     rx_tx_pps_minimum: Decimal = Decimal(0)
     rx_tx_pps_average: Decimal = Decimal(0)
     rx_tx_pps_maximum: Decimal = Decimal(0)
+
+
+@dataclass
+class NetworkNtttcpTCPPerformanceMessage(PerfMessage):
+    test_case_name: str = ""
+    test_date: datetime = datetime.utcnow()
+    platform: str = ""
+    location: str = ""
+    host_version: str = ""
+    guest_os_type: str = "Linux"
+    distro_version: str = ""
+    vmsize: str = ""
+    kernel_version: str = ""
+    lis_version: str = ""
+    ip_version: str = "IPv4"
+    protocol_type: str = "TCP"
+    data_path: str = ""
+    connections_num: int = 0
+    throughput_in_gbps: Decimal = Decimal(0)
+    latency_us: Decimal = Decimal(0)
+    buffer_size: Decimal = Decimal(0)
+    tx_packets: Decimal = Decimal(0)
+    rx_packets: Decimal = Decimal(0)
+    pkts_interrupts: Decimal = Decimal(0)
+    number_of_receivers: int = 1
+    number_of_senders: int = 1
+    sender_cycles_per_byte: Decimal = Decimal(0)
+    connections_created_time: int = 0
+    retrans_segments: int = 0
+    receiver_cycles_rer_byte: Decimal = Decimal(0)
+
+
+@dataclass
+class NetworkNtttcpUDPPerformanceMessage(PerfMessage):
+    test_case_name: str = ""
+    test_date: datetime = datetime.utcnow()
+    platform: str = ""
+    location: str = ""
+    host_version: str = ""
+    guest_os_type: str = "Linux"
+    distro_version: str = ""
+    vmsize: str = ""
+    kernel_version: str = ""
+    lis_version: str = ""
+    ip_version: str = "IPv4"
+    protocol_type: str = "UDP"
+    data_path: str = ""
+    connections_num: int = 0
+    number_of_receivers: int = 1
+    number_of_senders: int = 1
+    connections_created_time: int = 0
+    receiver_cycles_rer_byte: Decimal = Decimal(0)
+    send_buffer_size: Decimal = Decimal(0)
+    tx_throughput_in_gbps: Decimal = Decimal(0)
+    rx_throughput_in_gbps: Decimal = Decimal(0)
+    data_loss: Decimal = Decimal(0)
+    packet_size_kbytes: Decimal = Decimal(0)
+
+
+@dataclass
+class NetworkIperfSingleTCPPerformanceMessage(PerfMessage):
```

Comment:
Can it be merged with ntttcp? We can add the tool name as a field.

Reply:
they didn't use the same table

Reply:
It's ok, we can create a new table. It helps us compare results from different tools.

Reply:
add tool field and merge the NetworkIperfSingleTCPPerformanceMessage into NetworkTCPPerformanceMessage